.changeset/stupid-glasses-switch.md

```diff
@@ -0,0 +1,40 @@
+---
+'astro': minor
+---
+
+Adds a new virtual module called `astro:config` as part of the relative [RFC Stage 3](https://github.com/withastro/roadmap/pull/1106).
```

Comment:
```suggestion
Adds a new experimental virtual module `astro:config` that exposes a type-safe subset of your `astro.config.mjs` configuration
```
Can probably combine with the sentence below, and leave the links for the bottom so peple don't immediately click away 😄 ?